packages/wallets/src/features/accounts/modules/Poa/Poa.tsx

```diff
@@ -27,6 +41,28 @@ const Poa: React.FC<TPoaProps> = ({ onCompletion }) => {
     } = usePoa();
     const [errorDocumentUpload, setErrorDocumentUpload] = useState<THooks.DocumentUpload['error']>();
     const [showLoader, setShowLoader] = useState(true);
+    const [step, setStep] = useState<{ id: number; text: string }>({ id: 1, text: localize('Enter your address') });
```

Comment:
@likhith-deriv, I don't like the idea of using a step state to handle different screens. Instead of sharing the `<ModalStepWrapper />` for all the steps, we can create each step with its own `<ModalStepWrapper />`.

Reply:
Refactored the code 👍